llvm/lib/IR/DIBuilder.cpp

```diff
@@ -617,6 +617,29 @@ DIBuilder::createArrayType(uint64_t Size, uint32_t AlignInBits, DIType *Ty,
   return R;
```

Comment:
Could this function be implemented in terms of the new more general one?

Reply:
Yep.  I'll send it once I've rebuilt etc.